applications/templates/kube-system/02-appproject.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I propose to rename it like the [guestbook](https://github.com/edgelevel/gitops-k8s/blob/master/applications/templates/guestbook/project.yaml) example
* app prefix is redundant and confusing with `application.yaml`
* no number
* i.e. `project.yaml`

Reply:
👍